entry_types/paged/packages/pageflow-paged/src/frontend/ChapterFilter.js

```diff
@@ -1,5 +1,5 @@
 import _ from 'underscore';
-import {Object} from 'pageflow/frontend'
+import {default as Object} from '../Object'
```

Comment:
This is the same as `import Object from`. The `as Object` part is only needed when re-exporting (see index.js) to give the export a name. I'd also place the file in the `frontend` directory since it is not shared with `editor`.

Reply:
But `Object.js` does not export `Object`, only `BaseObject`. Therefore it's exported `{default as Object}` in `pagflow/ui` and `pageflow-paged` and imported `{default as Object}` here.

Reply:
I'll move it to frontend, though

Reply:
Since it's a default export the name doesn't matter. The import alone determines the name.

Reply:
I see, omitting the curly braces makes all the difference